### PR TITLE
Feature/final fixes

### PR DESCRIPTION
**Pull request by JeffKersting**
#### What does  this PR do?

Removes final code extraneous comments. Fixes minor styling issues. Adds README doc with related images.

#### Where should the reviewer start?

Read the README. Review code.

#### How should this be manually tested?

Add multiple idea cards. Favorite idea cards. Comment on idea cards. Refresh page to check data persistence. Delete idea cards. Show favorited. Show comments. Search ideas.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -31,6 +33,26 @@ commentSubmit.addEventListener('click', submitComment);
 modal.addEventListener('click', deleteComment);
 
 // FUNCTIONS
+
```
Comment (DrewBradley) [inline src/main.js:36]: Function organization looks amazing! Great work!

File: styles.css [hunk 2]
```diff
@@ -156,7 +157,6 @@ button:disabled{
   justify-content: space-between;
   background-color: #1f1f3d;
   height: 2em;
-  /* width: auto; */
```
Comment (DrewBradley) [inline styles.css:159]: Thank you for making sure that all our comments were removed.

File: src/main.js [hunk 3]
```diff
@@ -31,6 +33,26 @@ commentSubmit.addEventListener('click', submitComment);
...
+        savedComments.push(new Comment(idea.id, idea.parentId, idea.text));
+      }
+    }
+    for (var i = 0; i < savedComments.length; i++){
+      addCommentToIdea(savedComments[i], savedComments[i].parentId);
+    };
+    showIdeas(savedIdeas.sort(compareId));
+  }
+};
+
```
Comment (colefiscus) [inline src/main.js:55]: I say good move moving this!

File: src/comments.js [hunk 4]
```diff
@@ -4,10 +4,12 @@ class Comment {
     this.parentId = parentId;
     this.text = text;
   }
+
```
Comment (DrewBradley) [inline src/comments.js:7]: Nice work! Very great utilization of all of our knowledge that we have acquired thus far. I am just floored by our progress as a group. I mean, you folks are just swell.

**Review by DrewBradley — APPROVED** [review]

File: src/main.js [hunk 5]
```diff
@@ -236,7 +239,7 @@ function deleteComment(event) {
     }
     modal.style.display = 'none'
   }
-}
+};
```
Comment (colefiscus) [inline src/main.js:242]: Good eye catching all of these.

**Review by colefiscus — APPROVED** [review]
Let's turn it in!

**Pull request merged**